forum/qa-plugin/user-activity-log/user-activity-log.php

```diff
@@ -0,0 +1,139 @@
+        {
+            $qa_content=qa_content_prepare();
+            $qa_content['title']= qa_lang_html('user-activity-log/title');
+            $qa_content['form']=array(
+                'tags' => 'method="post" action="user-activity-log-search"',
+                'style' => 'wide',
+                'title' => qa_lang_html('user-activity-log/formTitle'),
+                
+                'fields' => array(
+                    'query' => array(
+                        'label' => qa_lang_html('user-activity-log/condition'),
+                        'tags' => 'name="request" value="'.$this->request.'"',
+                    ),
+                    
+                    'filter' => array(
+                        'type' => 'select',
+                        'tags'=>'name=condition value="'.$this->condition.'"',
+                        'options' => [
+                                'username' => qa_lang('user-activity-log/username'),
+                                'type' => qa_lang('user-activity-log/eventType'),
+                        ]
+                        ,
+                        'label' => qa_lang_html('user-activity-log/filters'),
+                    ),
+
+                    'date' => array(
+                        'label' => qa_lang_html('user-activity-log/date'),
+                        'tags' => 'placeholder=YYYY-MM-DD name=date value="'.$this->date.'"',
+                    ),
+
+                    'fromOldest' => [
+                        'type' => 'select',
+                        'tags'=>'name=fromOldest value="'.$this->sortBy.'"',
+                        'options' => [
+                                0 => qa_lang('user-activity-log/oldest'),
+                                1 => qa_lang('user-activity-log/newest'),
+                        ]
+                        ,
+                        'label' => qa_lang_html('user-activity-log/from'),
+                    ],
+
+                    'resultsCount' => array(
+                        'label'=> qa_lang_html('user-activity-log/count'),
+                        'tags'=> 'name=resultsCount value="'.$this->resultsCount.'"',
+                    )
+                ),
+
+                'buttons' => array(
+                    'submit' => array(
+                        'tags' => 'name="search"',
+                        'label' => qa_lang_html('user-activity-log/search-label'),
+                        'value' => '1',
+                    ),
+                ),
+            );
+
+            if(qa_get_logged_in_level() >= QA_USER_LEVEL_MODERATOR){
+                $qa_content['form']['fields']['filter']['options']['ip'] = qa_lang_html('user-activity-log/ipAdress');
+            }
+            $qa_content['navigation']['sub']= qa_admin_sub_navigation();
+            $qa_content['custom_2'] = 
+            '<div class = "modal-content">
+                <h1>'.qa_lang_html('user-activity-log/EventModalHeader').'</h1>
+                <p>
```

Comment:
Upewnij się, czy przeglądarka nie tworzy pustego elementu `<p></p>`, ponieważ generalnie element `<table>` [nie powinien](https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element:concept-element-tag-omission) znajdować się wewnątrz `<p>`.